sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/functions/FunctionsTest.java

```diff
@@ -71,21 +71,6 @@ public void testLeft() {
                 \s""");
     }
 
-    @Test
```

Comment:
this test looks fine, why delete it?

Reply:
it is already here: https://github.com/feldera/feldera/blob/89056b4b292cb1d433ce19e8eba60c8f4a6aa633/sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/ArrayFunctionsTests.java#L11

Reply:
the just go ahead and merge it when it passes the CI